### PR TITLE
PD-372 - port integration tests from Node world

### PR DESCRIPTION
**Pull request by elisechant**
* adds karma test runner; adds browserify step to karma - todo - remove… it from gulp

### CODE REVIEW

File: spec/javascripts/Widgets/SparklineWidgetSpec.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import $ from 'jquery';
```
Comment (elisechant) [inline spec/javascripts/Widgets/SparklineWidgetSpec.js]: todo - jasmine-jquery

**Comment by tobyhede** [comment]
Cool

File: spec/javascripts/Widgets/HeroWidgetSpec.js [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
+
+
   describe('hero widget creation', () => {
     //let widget;
     //let data = [[{'x': new Date('2016-01'), 'y': 40},
     //            {'x': new Date('2016-02'), 'y': null},
     //            {'x': new Date('2016-03'), 'y': 0}]];
     //beforeAll(function() {
-    //
+    //   $('body').append('<div id="chart" style="width:200px"></div>');
```
Comment (elisechant) [inline spec/javascripts/Widgets/HeroWidgetSpec.js]: undo this

File: spec/javascripts/Widgets/SparklineWidgetSpec.js [hunk 3]
```diff
@@ -7,7 +8,12 @@ module.exports = function() {
     let data = [[{'x': new Date('2016-01'), 'y': 40},
                 {'x': new Date('2016-02'), 'y': null},
                 {'x': new Date('2016-03'), 'y': 0}]];
+
+
     beforeAll(function() {
+
+      $('body').append('<div id="chart" style="width:200px"></div>');
```
Comment (elisechant) [inline spec/javascripts/Widgets/SparklineWidgetSpec.js]: undo this

**Pull request merged**